### DISCUSSION TITLE
发生错误时只能重启对话吗？我希望有办法继续对话。

### DISCUSSION THREAD

**Discussion by sandan20001116** [timqian/openprompt.co#35] [Q&A]
发生错误时只能重启对话吗？我希望有办法继续对话。

**Comment by timqian** [comment]
@sandan20001116 请问具体发生了什么错误？不能继续对话是因为一直报错还是无法输入了？

如果是无法输入，应该是一个 bug

对于比较长的对话，有可能是达到了 api 的字数限制，我们后期会优化这一部分

**Comment by Tsangsolo** [comment]
目前很多对话只能对话三次到五次，然后就被限制了，能否不要限制对话次数？

**Reply by timqian** [reply]
主要是看聊天记录整体长度，为了记住上下文，我们需要把前面所有聊天记录全部发送给 API, 如果前面的聊天比较长，比较容易触发长度限制。

有没有错误的截图看一下？

我们后面会优化长对话的实现方式，有进展了会在这里同步

**Comment by Tsangsolo** [comment]
嗯，我发现是因为字数的原因，我给她上传了一份小说大纲，做了一个人设后她就卡壳了。能优化文字对话长度就好了，非常感谢您的回答。

**Reply by timqian** [reply]
嗯，类似这种长文本，我们后面是有计划优化的
一个是用类似这种工具: https://github.com/jerryjliu/llama_index
另外就是通过 plugin 方式支持
